File: SDKs/Aspose.Tasks_Cloud_SDK_for_Java/src/main/java/com/aspose/tasks/model/WorkingTime.java
package com.aspose.tasks.model;

public class WorkingTime {
  private String FromTime = null;
  private String ToTime = null;
  /**
         * getFromTime
         * Gets String
         * @return FromTime
         */
  public String getFromTime() {
    return FromTime;
  }

        /**
         * setFromTime
         * Sets String
         * @param FromTime String
         */
  public void setFromTime(String FromTime) {
    this.FromTime = FromTime;
  }

  /**
         * getToTime
         * Gets String
         * @return ToTime
         */
  public String getToTime() {
    return ToTime;
  }

        /**
         * setToTime
         * Sets String
         * @param ToTime String
         */
  public void setToTime(String ToTime) {
    this.ToTime = ToTime;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class WorkingTime {\n");
    sb.append("  FromTime: ").append(FromTime).append("\n");
    sb.append("  ToTime: ").append(ToTime).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks_Cloud_SDK_for_.NET/src/Com/Aspose/Tasks/Model/WorkingTime.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class WorkingTime {
    public DateTime FromTime { get; set; }

    public DateTime ToTime { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class WorkingTime {\n");
      sb.Append("  FromTime: ").Append(FromTime).Append("\n");
      sb.Append("  ToTime: ").Append(ToTime).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
